src/modules/form/components/DynamicField.tsx

```diff
@@ -245,7 +247,6 @@ const DynamicField: React.FC<DynamicFieldProps> = ({
               sx={{
                 width,
                 maxWidth: MAX_INPUT_AND_LABEL_WIDTH,
-                '.MuiInputBase-root': { maxWidth: MAX_INPUT_WIDTH },
```

Comment:
I removed this because it's not used -- the SSN component's input widths are set directly in the component